templates/workspace_services/guacamole/user_resources/guacamole-azure-linuxvm/porter.yaml

```diff
@@ -82,6 +82,23 @@ outputs:
       - install
       - start
       - stop
+      - reset_password
+  - name: vm_username
+    type: string
+    applyTo:
+      - reset_password
+  - name: vm_password_secret_name
+    type: string
+    applyTo:
+      - reset_password
+  - name: keyvault_name
+    type: string
+    applyTo:
+      - reset_password
+  - name: new_password
+    type: string
+    applyTo:
+      - reset_password
```

Comment:
I think this will be picked up in the RP response and persisted to the resource's item on Cosmos. Right?
If yes, is there anyway around this? I mean, not to output those here at the bundle level and use only internally. Just a bit concerned we expose too much information which I prefer not be there...

Reply:
Yep valid concern as discussed in the stand-up - I'm testing to see if handling all in a single script will negate this problem